### ISSUE TITLE
Some models and train scripts in the paer "Bag of Tricks ... with CNN" are not released?

### ISSUE THREAD

**Issue by WalterMa** [dmlc/gluon-cv#586]
In the paper [Bag of Tricks for Image Classification with Convolutional Neural Networks](https://arxiv.org/abs/1812.01187), it gives quite nice explainations and comparasions for a lot of tricks. And I want to try some of them in other datasets.

But I noticed some models are not released yet, espicially in the Faster RCNN series (with cosine, smooth...):
![image](https://user-images.githubusercontent.com/7560132/51436919-e0b14500-1cd0-11e9-8a6e-bfe2a030c9c0.png)

Also I cannot find related train scripts ( with cosine annealing and label smooth ) used for training those Faster RCNN model described in above paper.
Will these be released after?

**Comment by zhreshold** [comment]
Do you mean FRCNN models with pretrained resnet that is trained with (cosine, label smooth) tricks?

**Comment by WalterMa** [comment]
> Do you mean FRCNN models with pretrained resnet that is trained with (cosine, label smooth) tricks?

yes

**Comment by duyaoqiong** [comment]
when will the tricks release?

**Comment by zhang11wu4** [comment]
when will the tricks release?

**Comment by zhreshold** [comment]
Please refer to https://github.com/dmlc/gluon-cv/pull/634, feel free to discuss in the PR. I am cloing this now.
